ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -2871,3 +2871,30 @@ def test084_filter_by_changed_by(self):
             self.assertTrue(key_found)
             # close child table
             self.orders_page.close_child_table(source=results[i])
+
+    def test090_if_cancel_archive_order_no_order_suborder_analysis_will_archived(self):
+        """
+        [Archiving][MainOrder]Make sure that if user cancel archive order,
+        No order or suborders or analysis of the order will be archived
+        LIMS-5404
+        """
+        self.info('create order')
+        self.order_page.create_new_order(material_type='Raw Material', save=False)
```

Comment:
Why do you create new order using GUI then duplicate it?  Is it required in jira  ticket? 
You can select random order and filter with order no then archive it

Reply:
in the ticket it said in step 2 add order and in step 3 enter valid data